src/game/g_spawn.cpp

```diff
@@ -176,6 +177,11 @@ field_t fields[] = {
 
     {"cursorhint", FOFS(s.dmgFlags), F_CURSORHINT},
 
+    {"savelimit", FOFS(ftSavelimit), F_LSTRING},
+    {"noghost", FOFS(damage), F_INT},
+    {"teamjumpmode", FOFS(health), F_INT},
+    {"leader_only_message", FOFS(ftLeaderOnlyMsg), F_LSTRING},
```

Comment:
`savelimit` and `leader_only_message` don't actually work here, `F_LSTRING` is made to work on char pointers, not arrays.